ci-operator/step-registry/telcov10n/functional/cnf-ran/cnf-tests-sno-worker/telcov10n-functional-cnf-ran-cnf-tests-sno-worker-commands.sh

```
@@ -0,0 +1,193 @@
+    find "$directory" -type f | while IFS= read -r filename; do
+        if [[ $filename == *"secretsync-vault-source-path"* ]]; then
+          continue
+        fi
+        local content
+        content=$(cat "$filename")
+        local varname
+        varname=$(basename "${filename}")
+        # Check if content has newlines - if so, use literal block scalar (|)
+        if [[ "$content" == *$'\n'* ]]; then
+          echo "${varname}: |"
+          echo "$content" | sed 's/^/  /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)